src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/queues/TransactionClosePaymentRetryQueueConsumer.kt

```diff
@@ -137,11 +144,15 @@ class TransactionClosePaymentRetryQueueConsumer(
                 RuntimeException(
                   "Unexpected transactionAtPreviousStep: ${tx.transactionAtPreviousState}")
               }
-          mono { nodeService.closePayment(tx.transactionId.value, closureOutcome) }
+
+          mono {
+              nodeService.closePayment(
+                tx.transactionId.value, closureOutcome._1(), closureOutcome._2())
+            }
```

Comment:
Rename `closureOutcome`, and:
```suggestion
              val outcome = closureOutcome._1()
              val authorizationCode = closeureOutcome._2()
              nodeService.closePayment(
                tx.transactionId.value, outcome, authorizationCode)
            }
```

Reply:
did it!